### PR TITLE
Create a page to visualize Trello comments of a user

### PR DESCRIPTION
**Pull request by colingrodecoeur360**
Issue:
- #15

# Context 

Flavien introduced his _MAgic PERformance Review Method_ (MAPERM) in this course: https://team.360learning.com/course/play/60e4664e08a16a9045addcbd

Key part:

> The MAPERM’s secret is to make a detailed assessment of each interaction, even the most insignificant one. There are two steps to achieve this:
-list all interactions;
-for each interaction, assess it.

Unfortunately, Trello's interface makes it really difficult to gather all the messages of a given person during some time period.
I figured it would be useful to use the Trello API to do that and display the results in a basic table.

We already have a `github.io` page with 2 tools: https://360learning.github.io/index.html
- A tool to generate a signature for mails
- A tool to automate the setup of onboarding boards

In this PR, I add a 3rd one: the Trello comments dashboard.

image

# Changes

- Add a new `trelloCommentsDashboard` tool, drawing inspiration from [Adrien's Trello board building helper](https://360learning.github.io/trelloOnboardingBuilder.html)
- Create the main fields of the form (Trello identifier, auth token, the reviewee's Trello username)
- Create a function that fetches a user's comments via the Trello API
- Display the results in a table
- Add sorting on the table's columns
- Add options to refine the results (date range, number of results, exclude some boards)

### CODE REVIEW

**Comment by AdriAt360** [comment]
@colingrodecoeur360 that's nice 😍 
I like the UX, even though I think it can be improved (feel free to not do everything depending on the feasibility/effort)
- Start date/End date
  - I guess the main usage will be for perf reviews
  - would it be easy to automatically select the complete previous quarter instead of the last 90 days 
  - e.g. on April, 19, select "start date = 108" and "end date = 19" to match the range 2022-01-01 -> 2022-03-31
- sorting
  - is it easy to highlight the selected sorting?
  - e.g. below would be "sorted by descending date" (inspiring from https://unicode-table.com/en/sets/arrow-symbols/#up-arrows, but feel free to choose different icons)
image
- Truncate comments to N lines
  - lines can be huge
  - and I guess it would be more relevant to have similar sizing of the content
  - would it be easy to do "Truncate comments to X chars" instead (with a default to 250 maybe)
- Their username
  - I don't find the "their" very explicit, what would you think about "Colleague's username" maybe?
- I must admit I missed the subtitle at first, because it seems out of the page
image
- To get started, get your Trello API information and store carefully your identifier and authentication token.
  - it would be even better if it's be explicit that this has to be done only once and then the credentials can be stored and reused
  - this is my fault on this part as it wasn't explicit in the other tool (by the way, feel free to change the wording there as well 😇 )
  - by the way I added such information inside [the linked doc](https://docs.google.com/document/d/1HwaedNa861gkj93TaradW5n0MID8cbeamiU5SXCvX64/edit#heading=h.ijjo2dol5z6v) (and granted you edit permission if you want to update this doc)

**Review by clementbat — APPROVED** [review]
Nice, no more excuse for not having precise and rational feedback 😅

File: trelloCommentsDashboard.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>360Tools - Trello comments dashboard</title>
+        <link rel="stylesheet" type="text/css" href="./style/style.css">
+        <link rel="stylesheet" type="text/css" href="./style/trelloCommentsDashboard.css">
+    </head>
+    <body>
+        <div>
+            <div id="app">
```
Comment (raphael-nachar) [inline trelloCommentsDashboard.html]: Why is `<div id="app">` inside another `<div>`?  Is this `<div>` necessary?

**Review by raphael-nachar — APPROVED** [review]
@colingrodecoeur360 Everything is good for me, good job 👍
I just have one question

File: trelloCommentsDashboard.html (hunk 1 above)
Comment (colingrodecoeur360) [reply trelloCommentsDashboard.html]: @raphael-nachar Good catch! I only copied the html of the other tools, but that div is actually not necessary:

https://github.com/360Learning/360learning.github.io/pull/16/commits/22fd5d3f7dc3722d967259085ead5b7fc5c6c5d6?w=1

Comment (raphael-nachar) [reply trelloCommentsDashboard.html]: @colingrodecoeur360 Perfect 👍

**Pull request merged**